.github/coverage.yml

```diff
@@ -0,0 +1,9 @@
+ignore-non-existing: true
+
+excl-line: "unreachable!"
+ignore:
+  - "bench-vortex/*"
+  - "fuzz/*"
+  - "home/*"
```

Comment:
what the heck is `home` ?

Reply:
I think should be `/home` for whatever reason it holds the Cargo cache and gets included

Reply:
seems like ignoring /* or ../* actually cleans that up